sample_app/config/locales/en.yml

```diff
@@ -60,7 +60,26 @@ en:
       create: "Create my account"
     create:
       welcome: "Welcome to the Sample App!"
-      has_error: "Oh, can not create accout, there are something wrong!"
+      has_error: "Wup, there are something wrong, can not create accout,"
```

Comment:
Như này mới đúng chứ nhỉ? :D :P
`"Oops!! There are something wrong, can not create accout"`